database/mongodb/mongodb.go

```diff
@@ -3,6 +3,7 @@ package mongodb
 import (
 	"context"
 	"fmt"
+	"github.com/mongodb/mongo-go-driver/x/bsonx"
```

Comment:
Avoid using any unstable/experimental modules.
What was wrong w/ the previous approach of creating a document?